src/main/java/core/basesyntax/MyNode.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class MyNode<T> {
+    private T value;
+    private MyNode<T> prev;
+    private MyNode<T> next;
+
+    MyNode(T x) {
+        value = x;
+    }
+
+    public T getValue() {
+        return value;
+    }
+
+    public void setValue(T value) {
+        this.value = value;
+    }
+
+    public MyNode getPrev() {
+        return prev;
+    }
+
+    public void setPrev(MyNode prev) {
+        this.prev = prev;
+    }
+
+    public MyNode getNext() {
+        return next;
+    }
+
+    public void setNext(MyNode next) {
+        this.next = next;
+    }
```

Comment:
Avoid using getters and setters in the `MyNode` class as per the checklist: 'Don't use getters and setters in class Node.'